docs/content/reference/data_types/image.md

```diff
@@ -3,16 +3,44 @@ title: Image
 order: 20
 ---
 
+`Image` represents a 2D raster image with various pixel format. They are a special case of 2D [Tensor](tensor.md) with an optional 3rd dimension when multiple color channels are used. Image with 1 (grayscale), 3 (RGB), or 4 (RGBA) channels are supported. Color channel maybe represented by any of the common scalar datatypes: 
+
+- `uint8`, `uint16`, `uint32`, `uint64`: color channels in 0-`max_uint` sRGB gamma space, alpha in 0-`max_int` linear space
+- `float16`, `float32`, `float64`: color channels in the 0.0-1.0 sRGB gamma space, alpha in 0.0-1.0 linear space
+- `int8`, `int16`, `int32`, `int64`: signed integers are cast into their unsigned counterpart without clipping
+
+
 ## Components and APIs
-## Components and APIs
+
 Primary component: `tensor`
 
 Secondary components: `colorrgba`, `draw_order`
 
+Note: `colorrgba` is currently only supported for images (i.e. 2D tensor with optional 3rd dimension). Furthermore, only the spatial Space View is able to use the color component.
+
```

Comment:
I actually don't really understand this note

Reply:
Perhaps @jleibs can weigh in and expand on this formulation with `colorrgba`?

Reply:
The scoping to image isn't needed here since it's specifically on the image page (as separate from the tensor this was grouped with previously. Just changing this to a comment about multiplicative tint.